src/gui/sdlmain.cpp

```diff
@@ -419,6 +425,40 @@ void GFX_SetTitle(Bit32s cycles, int /*frameskip*/, bool paused)
 	SDL_SetWindowTitle(sdl.window, title);
 }
 
+#if SDL_VERSION_ATLEAST(2, 0, 5)
```

Comment:
(side-comment) It will be nice to be able to require 2.0.5+ up-front, and then not have these concerns in the code.  Looks like 2.0.5 came out almost 5 years ago:

![Screenshot at 2020-04-12 11-19-53](https://user-images.githubusercontent.com/1557255/79076539-c08fec00-7caf-11ea-96a0-b4992ea5f823.png)

At what point do you think we can require it? (maybe when Ubuntu 16.04 goes EOL?)

Reply:
Yes, when 16.04 will go EOL (at that point I'll switch to 18.04 LTS as the barebones minimum that we need to support and remove all pointless ifdefs; ATM 18.04 is at SDL 2.0.8).

*I'm not even Ubuntu user; we just need some specific rule to follow about backwards compatibility and following LTS releases seems to strike nice balance between our needs as developers and user needs; other good candidate to follow would be Debian Stable releases (which move somewhat faster than Ubuntu LTS nowadays)*.

Reply:
Tracking Ubuntu LTS and Debian stable feels like a sweet spot, as current software compatibility and online forums tend to track closely with this "stable band" of supported versions.